src/main/java/com/smartbear/postman/script/SetGlobalVariableCommand.java

```diff
@@ -64,10 +64,16 @@ public boolean validate() {
 
     @Override
     public Object execute() {
+        TestProperty property = null;
         String propertyName = StringUtils.unquote(variableName.trim());
+        String propertyValue = StringUtils.unquote(variableValue);
+
         if (!project.hasProperty(propertyName)) {
-            TestProperty property = project.addProperty(propertyName);
-            property.setValue(StringUtils.unquote(variableValue));
+            property = project.addProperty(propertyName);
+        }
+
+        if (property != null) {
+            property.setValue(propertyValue);
```

Comment:
I think here we should get property if it exists and then set value for it. It could be created earlier, and we still should set its value.